python/ray/rllib/optimizers/multi_gpu.py

```diff
@@ -26,12 +30,15 @@ class LocalMultiGPUOptimizer(PolicyOptimizer):
     the TFMultiGPUSupport API.
     """
 
-    def _init(self, sgd_batch_size=128, sgd_stepsize=5e-5, num_sgd_iter=10):
+    def _init(self, sgd_batch_size=128, sgd_stepsize=5e-5,
+              num_sgd_iter=10, init_extra_ops=False):
         assert isinstance(self.local_evaluator, TFMultiGPUSupport)
         self.batch_size = sgd_batch_size
         self.sgd_stepsize = sgd_stepsize
         self.num_sgd_iter = num_sgd_iter
         gpu_ids = ray.get_gpu_ids()
+        # TODO(rliaw): fix this hack!
+        gpu_ids = os.environ["CUDA_VISIBLE_DEVICES"].split(",")
```

Comment:
Remove this but we need to make sure GPUs are utilized for both py2.7 and py3.6 on the newest ray